tests/parser/syntax/test_constants.py

```diff
@@ -201,3 +201,10 @@ def deposit(deposit_input: Bytes[2048]):
 @pytest.mark.parametrize("good_code", valid_list)
 def test_constant_success(good_code):
     assert compiler.compile_code(good_code) is not None
+
+
+@pytest.mark.parametrize(
+    "vyper_contract", ["constants/test_constantGetter1_Success.vy"], indirect=True
+)
+def test_constant_success_2(vyper_contract):
+    assert vyper_contract.foo() == 123
```

Comment:
Oh, this is clever. Basically, reuse compiled "successful parsing" test cases to additionally perform other checks?

Reply:
Yes, that's correct. This is more deliberate as compared to automatically creating fixtures for "successful parsing" test cases (which I still cannot figure out, and on further thought could be more confusing if all these contracts appear in the global namespace).

Reply:
Would the array of filenames passed in create multiple parametrized tests?

Also, some of the folders get really deep. I wonder if it could just assume a local path e.g. `./test_constantGetter1_Success.vy`

Reply:
> Would the array of filenames passed in create multiple parametrized tests?

Yes, I have updated the example to better demonstrate this.

> Also, some of the folders get really deep. I wonder if it could just assume a local path e.g. ./test_constantGetter1_Success.vy

Do you mean that there will be some sort of collection process in the current directory and subdirectories to get the Vyper files based on the filename provided?